docs/new_instrument.rst

```diff
@@ -209,21 +234,46 @@ are defined as instrument attributes within the initalization method.
 Required Routines
 -----------------
 
-Three methods are required within a new instrument module to support pysat
-operations, with functionality to cover finding files, loading data from
-specified files, and downloading new files. While the methods below are
-sufficient to engage with pysat, additional optional methods are needed for
-full pysat support.
+Three methods are required within a new instrument module to support
+:py:mod:`pysat` operations, with functionality to cover finding files, loading
+data from specified files, and downloading new files. While the methods below
+are sufficient to engage with :py:mod:`pysat`, additional optional methods are
+needed for full :py:mod:`pysat` support.
+
+Note that these methods are not directly invoked by the user, but by
+:py:mod:`pysat` as needed in response to user inputs.
+
+
+init
```

Comment:
Is init required by a check in pysat?

Reply:
https://github.com/pysat/pysat/blob/7abd5539e2c8e7092e7658d336a792fcb99c46dc/pysat/_instrument.py#LL530C9-L532C46

It is required by pysat, but not required for the user to supply.